[pytorch][export] Move is_param and get_param out of exir and into export

JacobSzwejbka opened the pull request:
Summary: These doesn't feel edge specific so moving out of exir.

Test Plan: ci

Differential Revision: D48361384

angelayi reviewed (approved):
bump on lintrunner failure

JacobSzwejbka committed:
[pytorch][export] Move is_param and get_param out of exir and into export (#107264)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/107264

X-link: https://github.com/pytorch/executorch/pull/67

These doesn't feel edge specific so moving out of exir.

Test Plan: ci

Reviewed By: tugsbayasgalan

Differential Revision: D48361384

fbshipit-source-id: f6476234c16d69ca57e6c30a3e568d08ad5f8062

JacobSzwejbka commented:
@pytorchbot merge
